src/rendergraph/tools/generate_shaders_gl.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
```

Comment:
Please move this to the `src/tools` directory in the root of the repo where our other tools are placed already. IMHO it don't belong in a subdirectory deep in the source tree where it's hard to find .

Reply:
Rendergraph is completely self-contained (except for the include of util/assert.h), that’s why I moved it here. After all, the shaders themselves are also here. Anyway, I will move it back.